nationalarchives/ds-wagtail
# DSRC: Authors to People

## About these changes

This PR will convert the `authors` app, `AuthorPage` and `AuthorIndexPage` to `people`, `PersonPage` and `PeopleIndexPage`

This is the groundwork for the people pages work.

## How to check these changes

- Pull the branch
- Run `manage rename_app authors people`
- Run migrations
- Check `AuthorPage`s (now `PersonPage`s), etc. 

## Before assigning to reviewer, please make sure you have


## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)
